addons/logistics_wirecutter/functions/fnc_isFence.sqf

```diff
@@ -22,8 +22,8 @@ TRACE_1("Checking if fence",_object);
 private _configOf = configOf _object;
 if !(isNull _configOf) then {
     // Check for isFence entry since we have valid configOf
-    getNumber (_configOf >> QGVAR(isFence)) == 1 // return
-} else {
-    // Check the p3d name against list (in script_component.hpp)
-    (getModelInfo _object select 0) in FENCE_P3DS // return
+    if (getNumber (_configOf >> QGVAR(isFence)) == 1) exitWith { true };
```

Comment:
this exitWith is inside a if/then so it won't have any effect

Reply:
Been too long since I've SQF'd - I'll refactor so that it works properly.

Reply:
Been too long since I've written SQF - my fault. Will fix!

Reply:
Fixed now. I added brackets around the secondary condition to get lazy evaluation.